salt/client/__init__.py

```diff
@@ -188,15 +188,15 @@ def __read_master_key(self):
             key_user = key_user.replace('\\', '_')
         keyfile = os.path.join(self.opts['cachedir'],
                                '.{0}_key'.format(key_user))
-        # Make sure all key parent directories are accessible
```

Comment:
@vernondcole I spoke with @cachedout about this offline. I'd rather see a change like this in a different PR so we can evaluate it as its own change. Can you make a new PR with this change and once that has been evaluated/merged in, it can be included here? Thanks!